client/utils/account-fees.tsx

```diff
@@ -85,19 +85,36 @@ export const getCurrentBaseFee = (
 		: accountFees.base;
 };
 
+export const getTooltipAndPillBaseFee = (
+	accountFees: FeeStructure
+): BaseFee => {
+	const tooltipAndPillBaseFee = getCurrentBaseFee( accountFees );
+
+	if ( ! tooltipAndPillBaseFee.percentage_rate ) {
+		tooltipAndPillBaseFee.percentage_rate =
+			accountFees.base.percentage_rate;
+	}
+
+	if ( ! tooltipAndPillBaseFee.fixed_rate ) {
+		tooltipAndPillBaseFee.fixed_rate = accountFees.base.fixed_rate;
+	}
+
+	return tooltipAndPillBaseFee;
+};
+
 export const formatMethodFeesTooltip = (
 	accountFees: FeeStructure
 ): JSX.Element => {
 	if ( ! accountFees ) return <></>;
-	const currentBaseFee = getCurrentBaseFee( accountFees );
```

Comment:
I'd rather leave it as it was and not use the `getTooltipAndPillBaseFee`. Maybe it is just me, but makes things harder to understand :)